### DISCUSSION TITLE
The ability to flip the screen horizontally on android

### DISCUSSION THREAD

**Discussion by FAZ1K07** [stoatchat/discussions#649] [Feature Suggestions]
The android app has not been updated since 2022, are you planning to improve the app?

**Comment by DeclanChidlow** [comment]
As the currently available app on Android is a web app, it doesn't need updates like other apps do. This will change with the new native app that is currently in development.

You can read about it here:
https://revolt.chat/posts/revolt-on-the-go

**Reply by FAZ1K07** [reply]
And when will it be released?

**Reply by DeclanChidlow** [reply]
The post states that "It's still cooking in the oven, but we're planning to release it in the coming months. We'll be releasing it as a beta first, so you'll be able to try it out before it's released to the public."
